src/python/review/quality/model.py

```diff
@@ -39,13 +43,28 @@ def apply(self, value):
 
 
 class Quality:
-    def __init__(self, rules: List[Rule]):
+    def __init__(self, rules: List[Rule], penalty_coefficient: int = 0):
```

Comment:
i think it would be great if you add some description what is penalty_coefficient